flow/project.py

```diff
@@ -2019,44 +2041,32 @@ def _print_progress(x):
                                     raise RuntimeError(
                                         "Unable to parallelize execution due to a pickling "
                                         "error: {}.".format(error))
-                        label_results = list(tqdm(
-                            iterable=results,
-                            desc="Collecting job label info", total=len(jobs), file=err))
+                        label_results = _generate_results_with_tqdm(
+                                results, "Collecting job label info", len_itr=len(jobs))
```

Comment:
Doesn't the description need to be a keyword argument here?

Reply:
yes, thank you for this